File: R/tests/testthat/test_bart_par4_withintrial.R
context("Test bart_par4_withintrial")
library(hBayesDM)

test_that("Test bart_par4_withintrial", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bart_par4_withintrial(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_bart_par4_withintrial.py
import pytest

from hbayesdm.models import bart_par4_withintrial


def test_bart_par4_withintrial():
    _ = bart_par4_withintrial(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
